factcast-site/documentation-docsy/content/en/Setup/properties.md

```diff
@@ -53,21 +53,26 @@ description: Properties you can use to configure FactCast
 
 ### Snapshots
 
+#### InMem-Snapshots
+
+| Property                                      | Description                                                                                          | Default |
+| --------------------------------------------- | :--------------------------------------------------------------------------------------------------- | :------ |
+| factcast.local.mem.deleteSnapshotStaleForDays | min number of days a snapshot is kept even though it is not read anymore. Must be a positive number. | 90      |
+
+#### InMemAndDisk-Snapshots
+
+| Property                            | Description                                                                                                                                                                                           | Default        |
+| ----------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------- |
+| factcast.local.disk.pathToSnapshots | path to store the snapshots in the file system.                                                                                                                                                       | java.io.tmpdir |
+| factcast.local.disk.maxDiskSpace    | max disk space to be used by the SnapshotDiskRepository. The oldest Snapshots will start to be removed after reaching 90% of the allocated space, never reaching 100%. Use 0 for unlimited disk space | 0              |
+
```

Comment:
also this should change if we align it with https://github.com/factcast/factcast/issues/3035

Reply:
Same here